app/cli/cli.py

```diff
@@ -147,14 +160,15 @@ def prep(fingerprint):
 
 @image.command()
 @click.argument('fingerprint', nargs=1)
```

Comment:
This might have the same bug but I didn't have time to look into exactly what this option does

Reply:
That will be next.